backend/LoadCell.py

```diff
@@ -0,0 +1,104 @@
+from enum import Enum
+import matplotlib.pyplot as plt
+import json
+import numpy as np
+
+class CalLoadCell():
+    class State(Enum):
+        NORMAL = 0
+        STORE = 1
+        WAIT = 2
+
+    def __init__(self, name="DefaultLoadCell") -> None:
+        self.name = name
+        self.load_cell = 0
+        self.tare = 0
+        self.slope = 1
+        self.vol_offset = 0
+        self.cal = False
+        self.cal_points = {}
+        self.mass = 0
+        self.volts = 0
+        self.txtval = 0
+        self.current_state = self.State.NORMAL
+
+    def reset(self):
+        self.current_state = self.State.NORMAL
+
+    def _collect_point(self, voltage, mass):
+        self.current_state: self.State.NORMAL
+        if not self.cal_points:
+            self.cal_points = {0: voltage}
```

Comment:
I see in the line below you have `self.cal_points[mass] = voltage` so I am assuming the cal_points dictionary is using a key which is the mass and a value which is the respective voltage.

In this line what you are doing is saying that for mass 0 the voltage is voltage, even though we are still passing in the mass.

For python dictionaries, since you are already initializing it to an empty dictionary you don't need to check if the dictionary exists, you can simply add to it with `self.cal_points[mass] = voltage`.

Reply:
I made a similar, change integrating this, but in the list containing tuples so that we can take in multiple masses